pkg/controller/template/render.go

```diff
@@ -332,6 +334,13 @@ func skipMissing(key string) (interface{}, error) {
 
 func cloudProvider(cfg RenderConfig) (interface{}, error) {
 	if cfg.Infra.Status.PlatformStatus != nil {
+		external, err := cloudprovider.IsCloudProviderExternal(cfg.Infra.Status.Platform, cfg.FeatureGate) //nolint
+		if err != nil {
+			glog.Error(err)
```

Comment:
is there a reason you're not returning the error here?

Reply:
Yes, the error check is made solely for featureGate validation, and is backed by the filtering behavior, which will return `false`. We don't want to break existing `cloudProvider` workflow, if the featureGate configuration is invalid. This is temporary solution: https://github.com/openshift/library-go/pull/993/files#diff-478af36e9fb994fc80d37b7d2f6ae207c67d8c43b94f98f6ae3e420808958ba9R41 Here is the place where the error comes from, it could be safely ignored and only printed.

Reply:
It will result in `external` false, which will not break and reconfigure cluster.

Reply:
thanks!